packages/block-library/src/template-part/index.php

```diff
@@ -19,12 +19,7 @@ function render_block_core_template_part( $attributes ) {
 	$content          = null;
 	$area             = WP_TEMPLATE_PART_AREA_UNCATEGORIZED;
 
-	if ( ! empty( $attributes['postId'] ) && get_post_status( $attributes['postId'] ) ) {
-		$template_part_id = $attributes['postId'];
-		// If we have a post ID and the post exists, which means this template part
-		// is user-customized, render the corresponding post content.
-		$content = get_post( $attributes['postId'] )->post_content;
-	} elseif (
```

Comment:
Note - removing this section is unrelated to the error noted in the PR description.

This code block is no longer necessary since `postId` has not been a valid attribute of the template part block for months now and content is always found via the `slug`/`theme` attributes.